tests/Unit/SearchEngine/QueryTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
```

Comment:
https://github.com/oat-sa/extension-tao-advanced-search/pull/65/files#r966815211

```suggestion
<?php

/**
```